docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,2005 @@
+{
```

Comment:
<p>I can't tell if I've been staring at this for too long now but this function feels a lot more readable now. What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>You are right! To be honest, I almost gave up and I thought that was it, and reader need to force themself to work with it. That was a great lesson for me.... if something looks weird... it IS weird and should be handled!  (So...thank you!)</p><p> </p><p>I think some lines such as 115-117 seem to be repetitive... That is why I mentioned, in another comment, that I think returning numpy array in function <code>_find_discords</code> can help us reduce number of lines here in function <code>_discords</code></p>

Reply:
<p>Excellent! I think it's a good lesson to remember to not be married to an implementation and be willing to start-from-scratch and then simplify. What we had previously had a clear "code smell" to it. While this isn't exactly perfect, it's likely pretty close. To write sustainable code, we need to have a willingness to always make things simpler and more readable. As you can see, we've drastically changed the code but the output is still the same and the logic is very straightforward.</p>

Reply:
Yes... I can see it! That "willingness" says a lot! Thanks!